### ISSUE TITLE
Questions about the implementation details in the paper

### ISSUE THREAD

**Issue by celinechanny** [aiiu-lab/Naturalistic-Adversarial-Patch#1]
Hi, I've read your paper on this project and thanks for bringing us this wonderful work!
But I got some confusion about the details of the paper's implementation. Sorry to be a bother.
About **Table 1** in the paper, I'm wondering what exactly **the patch scale** was used when implementing the evaluations, and also, did they evaluated in the INRIAPerson **test** set?
<p align="center">
  <img width="800" src="https://user-images.githubusercontent.com/66893885/181878956-78a49b5b-e218-48c8-ad88-4bbd2373b1cf.png" />
</p>

I got this confusion since the Yolov4tiny's mAP doesn't match any in Table 6(They are evaluated in the test set according to the paper).
<p align="center">
  <img width=400" src="https://user-images.githubusercontent.com/66893885/181879785-68fd7e85-ced6-4d56-b0b5-d002d84fb1e3.png" />
</p>

It will be really appreciated if you could reply to the questions~
Best regards!

**Comment by Kungbohan** [comment]
Hello, thank you for contacting us.
We used patch scale=0.2 and INRIA test set when implementing the evaluations. (The training set was used to train the patch only.)
For the patch scale, you can refer to
https://gitlab.com/EAVISE/adversarial-yolo/-/blob/master/load_data.py
L134 class PatchTransformer(nn.Module):
We followed the setting of this paper (adversarial-yolo).
In addition, the patch in Table6 is not the same as the patch in Table1, so the mAP values are a little different (17.27 and 8.67).
Table6 just illustrates an ablation study for the patch scale.
Table1 shows the final optimal adversarial patch.
If you have any questions, please let me know.

**Comment by celinechanny** [comment]
Thanks a lot!
